### PR TITLE
Project setup: adding of folders and the main file

### PR DESCRIPTION
**Pull request by kilonzi**
Addition of the [bin, lib] folders and the main.rb file

### CODE REVIEW

**Review by uche-inyama — APPROVED** [review]
## A CHANGE REQUIRED

### Hi team! :wave: 

All the requirements for this mile-stone have been met; you are ready to move on to the next milestone. But, to approve this project, there should be linter checks, with successful validation.

- [x] Implement your stickler with a successful check and submit another form for a code review.

[**Note**] Branches should be created one at a time, as per mile-stone.

#### ``` Have a great day ``` :slightly_smiling_face: 

``` Inyama, Uchechukwu Henry```

File: .stickler.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 linters:
   rubocop:
     display_cop_names: true
+    fixer: true
```
Comment (samgaco) [inline .stickler.yml:4]: :+1:

**Review by samgaco — APPROVED** [review]
Everything good, you can merge to the development branch and submit the project completion form. Regards.

----------------

* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

**Comment by edemagbenyo** [comment]
Merging the project into dev

**Pull request merged**